src/contracts/PreAMMBatcher.sol

```diff
@@ -404,4 +404,39 @@ contract PreAMMBatcher {
         }
         return newOrders;
     }
+
+    enum Direction {Ascending, Descending}
+
+    function isSorted(Order[] memory orders, Direction direction)
```

Comment:
Its still not clear what the orders are sorted by, by volume? by limit price? I would have the function name include the sorting key.